### ISSUE TITLE
[BUG] Catch22 on multivariate time series extracts 22 features in total instead of 22 features per variable/column

### ISSUE THREAD

**Issue by Sinnaeve** [alan-turing-institute/sktime#1826]
**Describe the bug**

I tried to use Catch22 to extract features of multivariate time series.
I followed the example from main/examples/catch22.ipynb
Using the multivariate data from this example I have an input of 40 instannces each with 6 series of 100 data points. (40x6x100)
I would expect 22 features per univariate series so an output of 40 instances each with 6 sets of 22 features (40x6x22), but I have an output of size 40x22.
It's writed: "The default procedure will concatenate each column prior to transformation" in the notebook, does that mean that for each instances, the algorithm take the 6 series (for 6 dimensions/columns) and flatten them in a single 600 data point time series ?
Does the features extracted means something then ? From what I uderstand from catch22 paper, they test their feature extraction method on univariate time series, so I would expect the multivariate version of the implementation of catch22 to apply the feature extractions mehod to each of the 6 univariate time series individually and concatenate the result after. 
**To Reproduce**

```python
from sktime.datasets import load_basic_motions
from sktime.transformations.panel.catch22 import Catch22
BM_X_train, BM_y_train = load_basic_motions(split="train", return_X_y=True)
print(BM_X_train.shape, BM_X_train.iloc[0,0].shape)
c22_mv = Catch22()
c22_output = c22_mv.fit_transform(BM_X_train)
print(c22_output.shape, c22_output.iloc[0,0])
```

**Expected behavior**

I would expect the multivariate version of the implementation of catch22 to apply the feature extractions mehod to each of the 6 univariate time series individually and concatenate the result after.
Like if I use a column transformer on each columns individually:
```python
from sktime.transformations.panel.compose import ColumnTransformer
transformer = ColumnTransformer(
    transformers = [(str(i), Catch22(), [BM_X_train.columns[i]]) for i in range(BM_X_train.shape[1])]
)
c22_output = transformer.fit_transform(BM_X_train)
print(c22_output.shape, c22_output.iloc[0,0])
```
Althought different format of the outputs could work as discussed here: https://github.com/alan-turing-institute/sktime/issues/1719

**Additional context**

**Versions**
'0.9.0'
<details>
System:
    python: 3.8.12
Python dependencies:
          pip: 21.2.2
   setuptools: 58.0.4
      sklearn: 1.0.1
       sktime: 0.9.0
  statsmodels: 0.12.1
        numpy: 1.19.3
        scipy: 1.7.3
       Cython: None
       pandas: 1.3.5
   matplotlib: 3.5.1
       joblib: 1.1.0
        numba: 0.54.1
     pmdarima: None
      tsfresh: None

</details>

**Comment by fkiraly** [comment]
Yes, indeed this is an issue.

Related discussion here:
https://github.com/alan-turing-institute/sktime/issues/1719
https://github.com/alan-turing-institute/sktime/pull/1663 (on summary transformer)

**Comment by fkiraly** [comment]
@Sinnaeve, do you have a preference on which proposed format from #1719 should be used? Or another format?

**Comment by fkiraly** [comment]
> so I would expect the multivariate version of the implementation of catch22 to apply the feature extractions mehod to each of the 6 univariate time series individually and concatenate the result after.

Yes, that is also what I would expect, although *how* we concatenate is one of the open points here.

**Comment by MatthewMiddlehurst** [comment]
As mentioned in the above PR, concatenating each dimension together was the default by the original author. I agree that returning 22 features per dimension is a better way of doing it, however.

**Comment by TonyBagnall** [comment]
fixed by #1869
